scalafix-core/src/main/scala/scalafix/internal/util/PrettyType.scala

```diff
@@ -172,7 +173,7 @@ class PrettyType private (
         Nil,
         Pat.Var(Term.Name(info.displayName)) :: Nil,
         None,
-        q"???"
+        "???".parse[Term].get
```

Comment:
why did you have to change that? The `.get` is suspicious, what about
```suggestion
        Term.Name("???")
```